CAM_PosX: -4.5
CAM_PosY: 0.0
CAM_PosZ: 0.86
CAM_OrtX: 0.0
CAM_OrtY: 0.0
CAM_OrtZ: 1.0
PCL_X: 0.0
PCL_Y: 0.0
PCL_Z: 0.0
PCL_Roll: 0.0
PCL_Pitch: 0.0
PCL_Yaw: 0.0
